.github/workflows/build-matrix.yml

```
@@ -197,7 +197,9 @@ jobs:
       - name: Configure Radium
         run: |
           cd build/Radium-Engine
-          cmake ../../src/Radium-Engine -GNinja -DCMAKE_CXX_COMPILER=${{ matrix.config.cxx }} -DCMAKE_C_COMPILER=${{ matrix.config.cc }} -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DRADIUM_USE_DOUBLE=${{ matrix.precision.value }} -DCMAKE_INSTALL_PREFIX=../../install/ -DCMAKE_EXECUTE_PROCESS_COMMAND_ECHO=STDOUT -DRADIUM_UPDATE_VERSION=OFF -DRADIUM_EXTERNAL_CMAKE_INSTALL_MESSAGE=LAZY -DRADIUM_INSTALL_DOC=OFF -DRADIUM_ENABLE_TESTING=ON -DRADIUM_ENABLE_COVERAGE=${{ matrix.coverage }} -DEigen3_DIR=${{ env.ext-dir }}/share/eigen3/cmake/ -DOpenMesh_DIR=${{ env.ext-dir }}/share/OpenMesh/cmake/ -Dcpplocate_DIR=${{ env.ext-dir }}/share/cpplocate/ -Dnlohmann_json_DIR=${{ env.ext-dir }}/lib/cmake/nlohmann_json/ -Dglm_DIR=${{ env.ext-dir }}/lib/cmake/glm/ -Dglbinding_DIR=${{ env.ext-dir }}/share/glbinding/ -Dglobjects_DIR=${{ env.ext-dir }}/share/globjects/ -Dstb_DIR=${{ env.ext-dir }}/include/stb/ -Dassimp_DIR=${{ env.ext-dir }}/lib/cmake/assimp-5.0/ -Dtinyply_DIR=${{ env.ext-dir }}/lib/cmake/tinyply/ ${{ env.glfw-path }}
+          cmake ../../src/Radium-Engine -GNinja -DCMAKE_CXX_COMPILER=${{ matrix.config.cxx }} -DCMAKE_C_COMPILER=${{ matrix.config.cc }} -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -DRADIUM_USE_DOUBLE=${{ matrix.precision.value }} \
+          -DCMAKE_INSTALL_PREFIX=../../install/ -DCMAKE_EXECUTE_PROCESS_COMMAND_ECHO=STDOUT -DRADIUM_UPDATE_VERSION=OFF -DRADIUM_EXTERNAL_CMAKE_INSTALL_MESSAGE=LAZY -DRADIUM_INSTALL_DOC=OFF -DRADIUM_ENABLE_TESTING=ON \
```

Do we still need `RADIUM_EXTERNAL_CMAKE_INSTALL_MESSAGE` here ?
```suggestion
          -DCMAKE_INSTALL_PREFIX=../../install/ -DCMAKE_EXECUTE_PROCESS_COMMAND_ECHO=STDOUT -DRADIUM_UPDATE_VERSION=OFF -DRADIUM_INSTALL_DOC=OFF -DRADIUM_ENABLE_TESTING=ON \
```